applications/DeepSpeed-Chat/training/utils/data/data_utils.py

```diff
@@ -84,7 +92,8 @@ def get_raw_dataset_split_index(local_rank, output_path, dataset_name, seed,
                                 split_name, data_split, split_index,
                                 data_size):
     index_file_name = f"{output_path}/{dataset_name}_seed{seed}_{split_name}_{data_split}_{split_index}.npy"
-    if not os.path.isfile(index_file_name):
+    # reindex each time since json file is more likely get modified
```

Comment:
Better to rephrase this comment as "reindex each time when using local jsonfile since it's more likely to get modified"